docs/source/multidc/gke.md

```diff
@@ -0,0 +1,159 @@
+
+As an example, we walk through the process of creating and configuring GKE clusters in two distinct regions. The example can easily be built upon to create infrastructure tailored to your specific needs.
+For simplicity, throughout the document we will use a number predefined values, which are only exemplary and can be adjusted according to your needs.
+
+## Prerequisites
+
+To follow the below guide, you first need to install and configure the following tools that you will need to create and manage GCP and Kubernetes resources:
+- gcloud CLI - Google Cloud Command Line Interface, a command line tool for working with Google Cloud resources and services directly.
+- kubectl – A command line tool for working with Kubernetes clusters.
+
+See [Install the Google Cloud CLI](https://cloud.google.com/sdk/docs/install-sdk) in GCP documentation and [Install Tools](https://kubernetes.io/docs/tasks/tools/) in Kubernetes documentation for reference.
+
+## Create and configure a VPC network
+
+For the clusters to have Inter-Kubernetes networking, we will create a virtual network shared between all the instances, with dedicated subnets for each of the clusters. 
+To create the subnets manually, we will create the network in custom subnet mode.
+
+### Create the VPC network
+
+Run the below command to create the network:
+```console
+gcloud compute networks create scylladb --subnet-mode=custom
+```
+
+With the VPC network created, we will create a dedicated subnet with secondary CIDR ranges for their Pod and Service pools in each region which the clusters will reside in.
+
+### Create VPC network subnets
+
+To create a subnet for the first cluster in region `us-east1`, run the below command:
+```console
+gcloud compute networks subnets create scylladb-us-east1 \
+    --region us-east1 \
+    --network scylladb \
+    --range 10.0.0.0/20 \
+    --secondary-range cluster=10.1.0.0/16,services=10.2.0.0/20
+```
+
+To create a subnet for the second cluster in region `us-west1`, run the below command:
+```console
+gcloud compute networks subnets create scylladb-us-west1 \
+    --region us-west1 \
+    --network scylladb \
+    --range 172.16.0.0/20 \
+    --secondary-range cluster=172.17.0.0/16,services=172.18.0.0/20
+```
+
+``` warning::
+    It is required that the IPv4 address ranges of the subnets allocated for the GKE clusters do not overlap.
+```
+
+Refer to [Create a VPC-native cluster](https://cloud.google.com/kubernetes-engine/docs/how-to/alias-ips) and [Alias IP ranges](https://cloud.google.com/vpc/docs/alias-ip) in GKE documentation for more information about VPC native clusters and alias IP ranges.
+
+## Create GKE clusters
+
+With the VPC network created, we will create two VPC native GKE clusters in dedicated regions.
+
+We will need to set kubelet CPU Manager policy to static by providing a config file. Save the following configuration in a `systemconfig.yaml` file:
+```
+kubeletConfig:
+  cpuManagerPolicy: static
```

Comment:
The EKS doc didn't require it because it refers to the single DC EKS doc for other node pools, and there it's only set for Scylla nodes pool. In the GKE doc it's set for the entire cluster - I guess it could be moved to the node pool as well there. Anyway removed the mention from here.

Reply:
I would rather have this mentioned here and fix EKS. It's important to have it if someone chooses Scylla. Without it, shard-per-core architecture doesn't provide it's benefits. People tend to stick with examples provided in docs.

Reply:
The thing is I don't really want to explain what this is about here. I'm much more in favor of creating references across the docs rather than repeating stuff - it's easier to keep track of and update that way. I'll create an issue to move it to scylla-pool creation (instead of cluster creation) in GKE doc. Then for both  GKE and EKS users will learn about it from single DC docs when creating scylla pools.

Reply:
https://github.com/scylladb/scylla-operator/issues/1485